token/services/db/sql/querybuilder.go

```diff
@@ -182,6 +182,27 @@ func certificationsQuerySql(ids []*token.ID) (string, []any, error) {
 	return builder.String(), tokenIDs, nil
 }
 
+func tokenRequestConditionsSql(params driver.QueryTokenRequestsParams) (string, []interface{}) {
```

Comment:
You can check in fsc `unversioned.go`
```
query := fmt.Sprintf("SELECT pkey, val FROM %s WHERE ns = $1 AND pkey IN %s", db.table, generateParamSet(2, len(keys)))
	logger.Debug(query, ns, keys)

	rows, err := db.readDB.Query(query, append([]any{ns}, castAny(keys)...)...)
	
```
We could extract these two methods to utils to re-use them

Reply:
sure, we can do that in another PR